test/integration/minishift.go

```diff
@@ -201,7 +202,7 @@ func (m *Minishift) setImageCaching(operation string) error {
 
 func (m *Minishift) imageExportShouldComplete(noOfImages int) error {
 	// poll till the output of the `minishift image list` shows number of cached images
-	timeout := time.NewTimer(20 * time.Minute)
+	timeout := time.NewTimer(30 * time.Minute)
```

Comment:
Why does it need 30 minutes as timeout?

Reply:
See @gbraad https://github.com/minishift/minishift/issues/1978#issue-294655158

Reply:
I was more interested in the why... especially as minikube 'can' work with a lower timeout

Reply:
Will do some investigation around it.

Reply:
@budhrg Any update on this?

Reply:
Not sure, how to go about the investigation. Any idea? 
What should I look for? Both b2d and minikube seems to have cache the images within 20 mins.

Reply:
In that case, let's move on. I am hesitant to have different timeouts. but perhaps this could be related to the startup time.